src/app/pcasts/controllers/listening_history_controller.py

```diff
@@ -15,8 +15,15 @@ def content(self, **kwargs):
       user = kwargs.get('user')
       offset = int(request.args['offset'])
       max_hs = int(request.args['max'])
+      dismissed = None
+      if 'dismissed' in request.args:
```

Comment:
This is starting to come up quite a bit in our code. Perhaps a built in solution to shorten this exists or we could create ours.